var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -400,7 +400,8 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
     # ###################### Dependencies ##########################
 
     # External Kokkos
-    depends_on("kokkos@4.2.01", when="@15.1.0: +kokkos")
+    depends_on("kokkos@4.3.00", when="@master: +kokkos")
```

Comment:
Do we know what the next trilinos release would be? This should probably be:
```suggestion
    depends_on("kokkos@4.3.00", when="@15.2: +kokkos")
```
which will include master *and* automatically ork with the next release.

(And do we really need to specify `.00`? will `.01` not work??)

Reply:
4.3.01 just doesn't exist yet but should work if it will ever exist.

Reply:
Great, in that case it should be 
```suggestion
    depends_on("kokkos@4.3", when="@15.2: +kokkos")
```